test/e2e-common.sh

```diff
@@ -181,6 +196,10 @@ function knative_teardown() {
     echo "Knative Build Pipeline YAML: ${INSTALL_PIPELINE_DIR}"
     echo ">> Bringing down Serving"
     ko delete --ignore-not-found=true -f "${INSTALL_RELEASE_YAML}" || return 1
+    if [[ -n "${INSTALL_MONITORING}" ]]; then
```

Comment:
Check for both flags here, since the YAML file can be set even if you don't want to install monitoring (default case).

Reply:
Updated.